### PR TITLE
Num 25 sort

### PR DESCRIPTION
**Pull request by attiks**

### CODE REVIEW

File: html/themes/custom/common_design_subtheme/css/styles.css [hunk 1]
```diff
@@ -67,3 +67,8 @@
   vertical-align: middle;
   background: transparent url("../components/rw-brand/rw-icons-sprite.svg") -84px -108px no-repeat;
 }
+
+.grid--locations .form-checkboxes {
```
Comment (rupl) [inline html/themes/custom/common_design_subtheme/css/styles.css:71]: I had to remove `.grid-locations` to get this to render. We're trying to style an instance of `paragraph--type-email-alerts`, right?

Comment (attiks) [reply html/themes/custom/common_design_subtheme/css/styles.css:71]: We are

Comment (attiks) [reply html/themes/custom/common_design_subtheme/css/styles.css:71]: I've pushed the missing file

Comment (rupl) [reply html/themes/custom/common_design_subtheme/css/styles.css:71]: I'm not a fan of Grid for alphabetical stuff but I'll follow-up later on.

**Review by rupl — APPROVED** [review]

**Pull request merged**